sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -191,22 +206,32 @@ internal class ViewAnnotationManagerImpl(
         )
       }
       if (viewAnnotation.handleVisibilityAutomatically) {
-        val isVisibleNow = (inflatedView.visibility == View.VISIBLE)
-        if (isVisibleNow == viewAnnotation.visible) {
+        val isAndroidViewVisible = (inflatedView.visibility == View.VISIBLE)
+        val isViewAnnotationVisible = viewAnnotation.visibility == ViewAnnotationVisibility.VISIBLE_AND_POSITIONED ||
```

Comment:
nit : create method 
`ViewAnnotation.isVisible = visibility == ViewAnnotationVisibility.VISIBLE_AND_POSITIONED || visibility == ViewAnnotationVisibility.VISIBLE_AND_NOT_POSITIONED`
used in several places

Reply:
created method for ViewAnnotation class instead of extension function